sieve_of_eratosthenes/sieveOfEratosthenes.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n) {
+	bool primes[n+1];
+	int i, sqrtOfn = sqrt(n)+1, j;
+	
+	memset(primes, true, sizeof(primes));
```

Comment:
but we have allotted memory to primes of `n+1` so sizeof(primes) gives byte memory size of the array while sizeof(bool*) will give 1 byte. 
Actually what should have been here is `sizeof(primes) * sizeof(bool*)` but as sizeof(bool*) is 1 it didn't effect anything.